sunbeam/eclipse_state.cpp

```diff
@@ -83,18 +85,18 @@ namespace {
 
 }
 
-void sunbeam::export_EclipseState() {
+void sunbeam::export_EclipseState(py::module& module) {
 
-    py::class_< EclipseState >( "EclipseState", py::no_init )
-        .add_property( "title", &EclipseState::getTitle )
-        .def( "_schedule",      &EclipseState::getSchedule,     ref() )
-        .def( "_props",         &EclipseState::get3DProperties, ref() )
-        .def( "_grid",          &EclipseState::getInputGrid,    ref() )
-        .def( "_cfg",           &EclipseState::cfg,             ref() )
-        .def( "_tables",        &EclipseState::getTableManager, ref() )
+    py::class_< EclipseState >( module, "EclipseState" )
+        .def_property_readonly( "title", &EclipseState::getTitle )
+        .def( "_schedule",      &EclipseState::getSchedule)
```

Comment:
These seem to return copies now, rather than references. Is that intentional?

Reply:
To be quite honest - going through the memory model with respect to references / copies / parents and what not was still on my list; that also includes running Valgrind.

Reply:
OK - this has been fixed.